Life-Pill/pharmacy-pos-frontend
# Refactor Cashier Management UI and Routing

## Pull Request: Refactor Cashier Management UI and Routing

### Changes Made
- Refactored routing paths and components for improved organization in the cashier management section.
- Created a proper routing structure and basic layout for items management using tables.
- Designed and implemented the UI for the new item adding information gathering page.
- Completed the UI design for the "Add Items" page.
- Updated the UI design for the "Update Items" page.

### Manual Testing Done
1. Tested the navigation paths to ensure proper routing between different pages in the cashier management section.
2. Checked the functionality of the items management layout, ensuring that tables display data correctly.
3. Verified the usability and appearance of the new item adding information gathering page.
4. Tested the "Add Items" page thoroughly, including uploading images, inputting item details, and handling form submissions.
5. Tested the "Update Items" page to ensure that it allows for editing and updating existing item information.

### Screenshot of Changes Made
![image](https://github.com/Life-Pill/pharmacy-pos-frontend/assets/119271523/204ec727-c356-440e-9efe-911d9a353b26)
![image](https://github.com/Life-Pill/pharmacy-pos-frontend/assets/119271523/c5570c93-9b72-4fe1-a610-71cbd1b0a7ef)

### Contributor
- **Name:** Tharindu Jayawardhana
- **Position:** Frontend Developer at LifePill

### Additional Notes
- The refactor improves code organization and enhances the user experience in the cashier management section.
- All changes have been thoroughly tested to ensure functionality and responsiveness.
- Ready for review and deployment.
